applications/nrf5340_audio/README.rst

```diff
@@ -761,14 +761,17 @@ Configuration
 Setting up the nRF5340 Audio repositories
 =========================================
 
-The application relies on the following :ref:`external OSS repositories <dm_code_base>` that need to be pulled using west:
+The application relies on the following repositories that need to be pulled using west:
 
-* LC3 software codec repository
-* Hardware codec driver repository
+* LC3 software codec repository - The default software codec for the application, which is not open-source.
+  The application must use the LC3 software.
```

Comment:
```suggestion
  The application only supports the LC3 codec by default.
```

Reply:
Rephrased a bit, since we use "default" in the previous line.